tests/spec_const/spec_const_exceptions_throwing_common.h

```diff
@@ -3,21 +3,22 @@
 //  SYCL 2020 Conformance Test Suite
 //
 //  Common code for expected exceptions throwing by specialization constants.
-//  In this tests we check that exception with code sycl::errc::invalid is thrown
+//  In this tests we check that exception with code sycl::errc::invalid is
+thrown
```

Comment:
I suggest using clang-format for code formatting. Looks like `//` is missing here.

Reply:
Do correct me if I'm wrong, (I'm still fairly new to clang-format), but this formatting mishap (like all of the following ones) are a result of applying clang-format on the files. I have VS Code configured to run clang-format on save. Applying the `.clang-format` file found in the repo puts "thrown" on the next line without a leading `//`. If that is an issue, someone should look into the `.clang-format` file. I checked being super specific:
```
C:\Kellekek\LLVM\15.0.3\bin\clang-format.exe --style=file:C:\Users\mate\Source\Repos\sycl-cts\.clang-format -i .\tests\spec_const\spec_const_exceptions_throwing_common.h
```
and it redoes this change.

Reply:
It's because the whole thing is already in a `/**/` comment block. It's strange, we might want to change that, but for now it's probably safer to add `//`

Reply:
> It's because the whole thing is already in a `/**/` comment block. It's strange, we might want to change that, but for now it's probably safer to add `//`

I didn't notice that. +1 for not mixing two commenting styles. It's very confusing.

Reply:
Fixed. There should be no mixed commenting styles anymore.